src/index.html

```diff
@@ -4,13 +4,64 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no,
+      initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Search bar Airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-bar__wrapper">
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        class="search-bar__form"
+      >
+        <label for="searchBarBig" class="search-bar__label big" data-qa="big">
+          <img
+            class="search-bar__img"
+            src="./images/Search.svg"
+            alt="Search Logo"
+          >
+        </label>
+        <input
+          type="search"
+          placeholder="Try “Los Angeles”"
+          autofocus="on"
+          autocomplete="off"
+          name="searchBarBigPanel"
+          class="search-bar__input big"
+          data-qa="big keypress"
```

Comment:
move `data-qa` with `big` value to your form tag, it's mentioned in readme:
- add attribute data-qa="big" for big search form, and data-qa="small" for small
- add attribute data-qa="keypress" to input in big search form